clients/cli/cmd/load.go

```diff
@@ -25,32 +26,26 @@ var loadCmd = &cobra.Command{
 		case "csv":
 			fileType = apiv1alpha.FileType_FILE_TYPE_CSV
 		default:
-			logAndQuitIfErrorExists(fmt.Errorf("unrecognized file type - must be one of 'parquet', 'csv'"))
+			utils.LogAndQuitIfErrorExists(fmt.Errorf("unrecognized file type - must be one of 'parquet', 'csv'"))
 		}
 		table := viper.GetString("table")
 		files := viper.GetStringSlice("file-path")
 
 		if len(files) == 0 {
-			logAndQuitIfErrorExists(fmt.Errorf("at least one `file` flag must be set"))
+			utils.LogAndQuitIfErrorExists(fmt.Errorf("at least one `file` flag must be set"))
 		}
 
 		apiClient := api.NewApiClient()
 		for _, file := range files {
 			err := apiClient.LoadFile(table, &apiv1alpha.FileInput{FileType: fileType, Uri: file})
-			logAndQuitIfErrorExists(err)
+			utils.LogAndQuitIfErrorExists(err)
 		}
 		log.Info().Msg("Success!")
 	},
 }
 
 func init() {
-	rootCmd.AddCommand(loadCmd)
-
-	loadCmd.Flags().String("file-type", "parquet", "(Optional) The type of file to load.  Either 'parquet' or 'csv'.")
-	loadCmd.Flags().String("table", "", "The table to load data into.")
-	loadCmd.Flags().String("file-path", "", "The path of the file to load on the Kaskada instance.")
-
-	viper.BindPFlag("file-type", loadCmd.Flags().Lookup("file-type"))
-	viper.BindPFlag("table", loadCmd.Flags().Lookup("table"))
-	viper.BindPFlag("file-path", loadCmd.Flags().Lookup("file-path"))
```

Comment:
viper is more for system configuration than for run-time use.  we should remove the viper usage inside most commands.